readme.md

```diff
@@ -66,15 +67,15 @@ When decoding the message, you need to know the number the original message was
 
 The `caesar()` function in the `src/caesar.js` file has three parameters:
 
-- _input_ refers to the inputted text to be encoded or decoded.
-- _shift_ refers to how much each letter is "shifted" by. A positive number means shifting to the right (i.e. "A" becomes "D") whereas a negative number means shifting to the left (i.e. "M" becomes "K").
-- _encode_ refers to whether you should encode or decode the message. By default it is set to `true`.
+- _input_ is a string that refers to the inputted text to be encoded or decoded.
+- _shift_ is an integer refers to how much each letter is "shifted" by. A positive number means shifting to the right (i.e. "A" becomes "D") whereas a negative number means shifting to the left (i.e. "M" becomes "K").
+- _encode_ is a boolean that refers to whether you should encode or decode the message. By default, it is set to `true`.
 
 When building the function, keep the following constraints and rules in mind:
 
 - If the `shift` value is not present, equal to `0`, less than `-25`, or greater than `25`, the function should return `false`.
-- Spaces should be maintained throughout, as should other non-alphabetic symbols.
-- Capital letters can be ignored.
+- Spaces in the message should be maintained before and after encoding or decoding, as should other non-alphabetic symbols.
+- Encoding is case-insensitive (e.g., both "a" or "A" would be encoded to the same result).
```

Comment:
Would love to know if I interpreted "Capital letters can be ignored' correctly with this change.